apps/back-office/src/app/(dashboard)/activity/page.tsx

```
@@ -0,0 +1,164 @@
+      location: "Lahore, Pakistan",
+      clientId: "1",
+      clientName: "Shawaiz Sarfraz",
+      type: "appointment",
+    },
+    {
+      date: "02/26/2025",
+      time: "10:45 AM (EST)",
+      event: "You updated client profile for John Smith",
+      ipAddress: "192.168.1.1",
+      location: "New York, USA",
+      clientId: "2",
```

Biome (1.9.4)

[error] This property value named clientId is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named clientId


(lint/suspicious/noDuplicateObjectKeys)